code_examples/core_features/run_core_features.ts

```diff
@@ -112,42 +113,17 @@ async function main(): Promise<void> {
 
   // These should not conflict anymore since all accounts are different.
   await Promise.all([
-    (async () => {
-      try {
-        await runAllClaiming(claimingTestAccount)
-      } catch (e) {
-        console.error('Claiming flow failed')
-        throw e
-      }
-    })(),
-    (async () => {
-      try {
-        await runAllDid(didTestAccount)
-      } catch (e) {
-        console.error('DID flow failed')
-        throw e
-      }
-    })(),
-    (async () => {
-      try {
-        await runAllWeb3(web3TestAccount)
-      } catch (e) {
-        console.error('Web3name flow failed')
-        throw e
-      }
-    })(),
-    (async () => {
-      try {
-        await runAllLinking(
-          nodeAddress,
-          accountLinkingTestAccount,
-          faucetAccount
-        )
-      } catch (e) {
-        console.error('Linking flow failed')
-        throw e
-      }
-    })()
+    runAllClaiming(claimingTestAccount).catch(() =>
```

Comment:
@arty-name what's your opinion about this?

Reply:
One thing that is definitely lost in this change is the re-throwing of errors. I assume it might have been important for detection of failures, in that case it should be restored. Let’s see how the code looks to you afterwards. Personally, I always prefer async/await to promises, also because with them you do not randomly lose re-throwing of errors ;)

Reply:
If the readability is too bad, the logging could be moved inside the functions themselves, I believe. This piece of code running things in parallel and logging at the same time is not particularly elegant in any form, I’d say.